### PR TITLE
Refer to packages instead of integrations

### PR DESCRIPTION
**Pull request by mtojek**
Fixes: https://github.com/elastic/elastic-package/issues/35

README preview: https://github.com/mtojek/elastic-package/blob/35-adjust-wording/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-The elastic-package is a command line tool, written in Go, used for developing Elastic Integrations.
+The elastic-package is a command line tool, written in Go, used for developing Elastic packages.
+
+*For experimental use only*
 
 ## Features
 
 TODO
 
+## Supported packages
```
Comment (masci) [inline README.md:11]: Let's do "package types" so the `type` key will sound more familiar if an user peeks into a manifest

File: cmd/testrunner.go [hunk 2]
```diff
@@ -9,8 +9,8 @@ import (
 func setupTestCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "test",
-		Short: "Run test suite for the integration",
-		Long:  "Use test runners to verify if the integration collect logs are metrics properly.",
+		Short: "Run test suite for the package",
+		Long:  "Use test runners to verify if the package collects logs are metrics properly.",
```
Comment (masci) [inline cmd/testrunner.go:13]: typo: s/`logs are metrics`/`logs and metrics`

File: README.md (hunk 1 above)
Comment (mtojek) [reply README.md]: Agree.

Changed to "Supported package types".

File: cmd/testrunner.go (hunk 2 above)
Comment (mtojek) [reply cmd/testrunner.go]: Fixed.

**Pull request merged**